README.md

```diff
@@ -1 +1,28 @@
-# swift-differentiation-testing
\ No newline at end of file
+# swift-differentiation-testing
+This repository serves as a test suite of crashes and other broken behaviour related to the [Differentiable Swift](https://github.com/differentiable-swift#meet-differentiable-swift) language feature. It aims to provide a low bar for Differentiable Swift users to provide reproducers to crashes or incorrect behaviour. 
+Tests that reproduce currently broken behaviour are added as expecting to fail and once fixed in future toolchain releases will be marked as fixed and used to run as a test suite. 
+The tests are currently divided into 4 different categories: 
+- Compiletime crash tests, checks for unexpected while compiling code.
+- Runtime crash tests, checks for unexpected crashes while running code.
+- Runtime validation tests, checks for incorrect results while running code.
+- Runtime performance tests, checks for changes in performance as implementations evolve.
+
+The ultimate goal of this repository is for the library portion to no longer exist since all implementations are either upstreamed to [Swift](https://github.com/swiftlang/swift) or the provided workarounds are no longer needed since there's more direct language support making them obsolete. 
+We would ultimately like this repository to become a collection of Examples and documentation on the language feature so that people can more easily get started using Differentiable Swift! 
+
+## Overview
+Currently this repository provides the following:
+- [Compiletime crash test](CompiletimeCrashTests) suite
+- [Runtime crash test](RuntimeCrashTests) suite
+- [Runtime validation test](RuntimeValidationTests) suite
+- [Runtime performance test](RuntimePerformanceTests) suite with [results being tracked over time](https://differentiable-swift.github.io/swift-differentiation-testing/RuntimePerformanceTests/results/)
```

Comment:
I think we should make this link easier to find. It being here is OK bit it might also be nice to have in the "About" sidebar of the repo